ffi/diplomat/src/properties_sets.rs

```diff
@@ -31,14 +60,776 @@ pub mod ffi {
                 .map_err(Into::into)
                 .into()
         }
+        /// Gets a set for Unicode property alnum from a [`ICU4XDataProvider`].
+        #[diplomat::rust_link(icu::properties::sets::load_alnum, Fn)]
+        pub fn try_get_alnum(
```

Comment:
Question: Why `try_get` instead of `load` like in Rust?

Reply:
It's the naming scheme we had already.

Reply:
I prefer `load` and using this opportunity to fix the pre-existing ones.